content/docs/reference/pkg/python/pulumi_kafka/_index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Package pulumi_kafka
+linktitle: pulumi_kafka
+notitle: true
+---
+
+<div class="section" id="pulumi-kafka">
+<h1>Pulumi Kafka<a class="headerlink" href="#pulumi-kafka" title="Permalink to this headline">¶</a></h1>
+<blockquote>
+<div><p>This provider is a derived work of the <a class="reference external" href="https://github.com/terraform-providers/terraform-provider-kafka">Terraform Provider</a> distributed under
+<a class="reference external" href="https://www.mozilla.org/en-US/MPL/2.0/">MPL 2.0</a>. If you encounter a bug or missing feature, first check the
+<a class="reference external" href="https://github.com/pulumi/pulumi-kafka/issues">pulumi/pulumi-kafka repo</a>; however, if that doesn’t turn up
+anything, please consult the source <a class="reference external" href="https://github.com/terraform-providers/terraform-provider-kafka/issues">terraform-providers/terraform-provider-kafka repo</a>.</p>
+</div></blockquote>
```

Comment:
Unfortunately, on the Python side, this text doesn't come from a generated README file in the provider. It's specified in the following templates:
- https://github.com/pulumi/docs/blob/071890055244447abf9c24b79bf1e47103e9b00b/tools/pydocgen/pydocgen/templates/providers/provider_with_module.rst
- https://github.com/pulumi/docs/blob/071890055244447abf9c24b79bf1e47103e9b00b/tools/pydocgen/pydocgen/templates/providers/provider_without_module.rst

We'll need to make some changes to the way we generate the Python docs to allow pointing to the right source repo.

OR... (or in the meantime)... in this PR, modify the changes to `tools/pydocgen/pulumi-docs.json` to make `terraform_provider_name` an empty string instead of `"kafka"`. Then this text shouldn't show up in the generated Python docs at all. Not having the text is probably better than having it pointing to a non-existent repo.